moduleroot/.rubocop.yml.erb

```
@@ -2,6 +2,9 @@
 # Managed by modulesync - DO NOT EDIT
 # https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
+<% if @configs['use_rubocop_todo'] -%>
```

I think you can make it dynamic, depending on whether the file exists:
```suggestion
<% if File.exist?(File.join(@metadata[:workdir], '.rubocop_todo.yml')) -%>
```